python/paddle/v2/framework/tests/test_conv2d_op.py

```diff
@@ -91,30 +76,47 @@ def test_check_grad_no_input(self):
             max_relative_error=0.05,
             no_grad_set=set(['Input']))
 
-    def init_groups(self):
+    def init_test_case(self):
+        self.groups = 1
```

Comment:
init_group will be covered by this code.

Reply:
Fixed